babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  presets: [['@shopify/babel-preset', {typescript: true}]],
```

Comment:
A lack of `react: true` seems suprising here - I'd have thought we'd need to parse jsx so things like storybook would work

Though I note that the path in https://github.com/Shopify/polaris/blob/main/polaris-react/.storybook/babel.config.js#L1 is now incorrect and should be updated.

You should check that storybook continues to work